SYCL/ESIMD/Stencil.cpp

```diff
@@ -79,7 +72,13 @@ bool CheckResults(float *out, float *in) {
   return true;
 }
 
-int main(void) {
+int main(int argc, char *argv[]) {
+  if (argc != 2) {
```

Comment:
I suggest to set DIM_SIZE default to 10 and override if argc == 2.

Reply:
... so that it is runnable w/o args, which most users will have no idea how to set.

Reply:
Done